Tests/TestingMacrosTests/ConditionMacroTests.swift

```diff
@@ -16,7 +16,7 @@ import SwiftOperators
 import SwiftParser
 public import SwiftSyntax
 import SwiftSyntaxBuilder
-import SwiftSyntaxMacros
+public import SwiftSyntaxMacros
```

Comment:
I know why we need `public import` in `TestingMacros`, but why do we need it in any files in `TestingMacrosTests`?

Reply:
Static linkage on some platforms may cause conflicts, so best to just be consistent.